repository-data/webfiles/src/main/resources/site/freemarker/hee/catalog/report-main.ftl

```diff
@@ -76,6 +102,64 @@
+                                        <strong>Published: </strong> ${document.publicationDate.getTime()?date?string["${datePattern}"]}
+                                    </div>
+                                    <div class="heeuk-link-inactive-state-s"><strong>Updated: </strong> ${document.properties['hippostdpubwf:lastModificationDate'].getTime()?date?string["${datePattern}"]}</div>
+                                    <#if landingPage??>
+                                        <div class="heeuk-link-inactive-state-s">
+                                            <strong>Topics: </strong>
+                                            ${landingPage.publicationType}
+                                            <#if landingPage.publicationProfessions?has_content>
+                                                <#list landingPage.publicationProfessions as profession>
+                                                    - ${profession}
+                                                </#list>
+                                            </#if>
+                                            <#if landingPage.publicationTopics?has_content>
+                                                <#list landingPage.publicationTopics as topic>
+                                                    - ${topic}
+                                                </#list>
+                                            </#if>
+                                        </div>
+                                        <strong>Estimated reading time:</strong> ${landingPage.readTime} min
+                                    </#if>
+                                </div>
+                            </div>
+                            <#if landingPage?? && landingPage.documentVersions?has_content>
+                                <div class="nhsuk-card">
+                                    <div class="nhsuk-card__content">
+                                        <h3 class="nhsuk-heading-m">Alternative versions</h3>
+                                        <ul class="nhsuk-resources__list">
+                                            <#list landingPage.documentVersions as link>
+                                                <#if link?? && link.mimeType != 'application/vnd.hippo.blank'>
+                                                    <@docDetailBlockForDocLink docLink=link/>
+                                                </#if>
+                                            </#list>
+                                        </ul>
+                                    </div>
+                                </div>
+                            </#if>
+                            <#if landingPage?? && landingPage.languageVersions?has_content>
+                                <div class="nhsuk-card">
+                                    <div class="nhsuk-card__content">
+                                        <h3 class="nhsuk-heading-m">Languages</h3>
```

Comment:
Could we handle empty `Languages` boxes (when documents aren't available) ?

<img width="334" alt="Screenshot 2022-12-16 at 08 01 09" src="https://user-images.githubusercontent.com/35143542/208053487-2db75648-edab-48e8-8891-4ce04ace6bd2.png">